.github/workflows/lint.yml

```diff
@@ -21,7 +21,8 @@ jobs:
       run: pip install black
```

Comment:
Does `super-linter` require `black` to be manually installed?

Reply:
Let's try without it...